observability/README.md

```diff
@@ -206,10 +213,17 @@ In this quickstart Zipkin is used for tracing. Examine [*./deploy/zipkin.yaml*](
 
 Deploy Zipkin to your cluster by running:
 
+<!-- STEP
+name: Apply tracting config
+expected_stdout_lines: 
```

Comment:
shouldn't there be some output to verify that zipkin is applied successfully ?

Reply:
Oh sure.